### ISSUE TITLE
missing files 

### ISSUE THREAD

**Issue by firaskhemakhem** [youqingxiaozhua/APViT#4]
I can't figure out where to get the 'ir.pdparams' and 'vit.pdparams' files in order to rebuild the model

**Comment by youqingxiaozhua** [comment]
Sorry for the confusion. These two param files are pre-trained weights for the IR-50 and ViT-Small models. You can download them following the [Readme](https://github.com/youqingxiaozhua/APViT#installation) and convert them to paddle with the [x2paddle tool](https://github.com/PaddlePaddle/x2paddle#pytorch%E6%A8%A1%E5%9E%8B%E8%BD%AC%E6%8D%A2). Notable, our paddle implementation is for inference only. Some crucial training details (e.g., data augmentations, learning rate scheduler) are not aligned with the paper.
